airbyte-webapp/src/packages/cloud/services/thirdParty/launchdarkly/LDExperimentService.tsx

```diff
@@ -123,7 +99,7 @@ const LDInitializationWrapper: React.FC<React.PropsWithChildren<{ apiKey: string
   };
 
   if (!ldClient.current) {
-    ldClient.current = LDClient.initialize(apiKey, mapUserToLDUser(user, locale, null));
+    ldClient.current = LDClient.initialize(apiKey, createUserContext(user, locale));
```

Comment:
I think we also should set the multi context here directly, so we don't have the initial auth against LD without the workspace context?

Reply:
Sure, we have access to `workspaceId` here on the first render, so we can conditionally add the workspace context. Added in 9360c9c8150242c7345362d066b97c7b3206529b